### PR TITLE
fix: remove namespace switch used for helm3 alpha

### PR DESCRIPTION
**Pull request by yelhouti**
Signed-off-by: Youssef El Houti <youssef.elhouti@gmail.com>

#### Submitter checklist

- [X] Change is code complete and matches issue description.
- [X] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/helm/helm_cli.go [hunk 1]
```diff
@@ -450,19 +430,6 @@ func (h *HelmCLI) Template(chart string, releaseName string, ns string, outDir s
 // UpgradeChart upgrades a helm chart according with given helm flags
 func (h *HelmCLI) UpgradeChart(chart string, releaseName string, ns string, version string, install bool, timeout int, force bool, wait bool, values []string, valueStrings []string, valueFiles []string, repo string, username string, password string) error {
 	var err error
-	currentNamespace := ""
-	if h.Binary == "helm3" {
```
Comment (hferentschik) [inline pkg/helm/helm_cli.go]: I like seeing the removal of these comparisons against "helm3". This looks promising. What issue/behaviour do you try to fix/implement with this change? Could you capture it in an issue?

Comment (yelhouti) [reply pkg/helm/helm_cli.go]: in helm3 alpha we used to switch the namespace manually in the kubeconfig file as the flag "--namespace" wasn't working, now that we use a later version in our container, there is no more need to modify the config. this basically juste a cleanup... not a real issue

**Comment by hferentschik** [comment]
@garethjevans, looking at the code, you seemed to have been the one putting these "helm3" checks in the first place. It seems odd to me that we still keep some in place and that we deal with a "helm3" binary in general. However, since it does not seem to break anything obvious I think this change should be fine. WDYT?

**Pull request merged**